modules/mobile/app/controllers/mobile/v0/decision_letters/downloads_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'claim_letters/claim_letter_downloader'
+
+module Mobile
+  module V0
+    module DecisionLetters
+      class DownloadsController < ApplicationController
+        before_action { authorize :bgs, :access? }
```

Comment:
I ended up not doing it because that policy checks for icn.present? && user.ssn.present? && user.participant_id.present? and the upstream service does some fallback logic if paripicant id is missing then it uses the ssn for file number. `lib/claim_letters/claim_letter_downloader.rb:54`